### DISCUSSION TITLE
Weblate glossary

### DISCUSSION THREAD

**Discussion by JunkFood02** [JunkFood02/Seal#206] [General]
I noticed there's a new component on Hosted Weblated called [glossary](https://hosted.weblate.org/translate/seal/glossary/en).
I'm not very familiar with how Weblate works. Is this component redundant or necessary?  (I see there're three yt-dlp's in it)

@chiyi4488 @comradekingu

**Comment by chiyi4488** [comment]
The glossary is necessary and it helps to avoid the use of different synonyms.

YT-DLP is my mistake, just leave one. (Leave the one with notes)

詞彙表是有必要的，它有助於統整詞彙，以避免同義詞氾濫使用。
YT-DLP 是我的失誤，只需留下一個有註釋的即可。
